### ISSUE TITLE
metrics of a resumed run does not line up

### ISSUE THREAD

**Issue by saswatac** [wandb/client#1040]
`wandb --version && python --version && uname`

* Weights and Biases version: wandb, version 0.8.36
* Python version: Python 3.6.10 :: Anaconda, Inc.
* Operating System: Linux

* Tensorflow version: 2.1.0

### Description

When I resume a run which was earlier killed, the metrics from the run do not line up correctly with previously emitted metrics.

State of metrics at the time the job was killed -

![image](https://user-images.githubusercontent.com/4872140/82082320-1c52f800-96e8-11ea-82c4-f067ca1527ac.png)

Metrics were logged at global steps 100, 200, 300, 400.

State of the metrics after the job resumed and completed -

![image](https://user-images.githubusercontent.com/4872140/82082451-52907780-96e8-11ea-9f7d-ceca06fa9325.png)

Metric when seen from tensorboard  -

<img width="382" alt="Screenshot 2020-05-14 at 19 54 25" src="https://user-images.githubusercontent.com/4872140/82082558-84094300-96e8-11ea-91ed-9c3f32dfedb4.png">

Metric when plotted against step instead of global step -

![image](https://user-images.githubusercontent.com/4872140/82082643-ad29d380-96e8-11ea-9a98-7c3a91793521.png)

I expected the metrics in wandb to be aligned when plotted against the global step as seen in tensorbpard.

### What I Did

wandb initialized in training script as follows -
```
wandb.init(
        project=project,
        config=config,
        name=name,
        dir=wandb_dir,
        sync_tensorboard=True,
        resume=True,
    )
```

Training ran for 400 steps, then the process was killed and exited with non zero status. Then the training was started again from the same directory.

**Comment by tyomhak** [comment]
Hi, thanks for the feedback, we will take a look at this!

**Comment by tyomhak** [comment]
Hey @saswatac  can you send us the log file from this run?

**Comment by ariG23498** [comment]
Hey @saswatac 
I have tried reproducing this ticket and have found that the resume feature does line up the steps.
To reproduce:
```python
run = wandb.init(entity="xxx",project="xxx",resume=True)

with run:
    for i in range(20):
        if i == 10:
            # Ctrl c
            sys.exit()
        run.log({"metric":i})
```
This executes a `Ctrl+C` at step 10. Later I run the same code again and this is the visualization that I see 👇 
![image](https://user-images.githubusercontent.com/36856589/108733378-770d6000-7554-11eb-870f-1d8daed585f6.png)
We need to log the metrics keeping in mind the crash that has happened. The user needs to take care of that and the steps are automatically lined up as shown.

Closing this issue 😄
